README.md

```diff
@@ -67,6 +67,36 @@ If you need to build your own AMI (because you've changed something in the `pack
 make packer
 ```
 
+## Support Policy
+
+We provide support for security and bug fixes on the current major release only.
+
+If there are any changes in the master branch since the last tagged release, we
+aim to publish a new tagged release of this template at the end of each month.
+
+### AWS Regions
+
+We support all AWS Regions, except China and US GovCloud.
+
+We aim to support new regions within one month of general availability.
+
+### Operating Systems
+
+We build and deploy the following AMIs to all our supported regions:
+
+- Amazon Linux 2 (x86_64)
+- Amazon Linux 2 (arm64)
+- Windows Server 2019 (x86_64)
+
+### Buildkite Agent
+
+The Elastic CI Stack template [published from the master branch](https://s3.amazonaws.com/buildkite-aws-stack/latest/aws-stack.yml)
+tracks the latest Buildkite Agent release.
```

Comment:
the only caveat with this is that the master branch template only works in us-east-1

Reply:
I looked into this and the [latest master build](https://buildkite.com/buildkite-aws-stack/buildkite-aws-stack/builds/2389#0cecf4c3-ebfb-4cfb-bfc0-bdde6b8ccad3) did copy them. I checked the [copy step](https://github.com/buildkite/elastic-ci-stack-for-aws/blob/master/.buildkite/steps/copy.sh#L109) and it does the copy for `master` or tagged builds 🥳

So branch builds don’t have their AMI copied but master and tags do, that’s what I would have done if it didn’t already work that way 😄

Reply:
Just tested deploying master to ap-southeast-2 too and it worked, hooray!

Reply:
oh, neat! Sorry for the bad advice 😳